bufferserver/src/test/java/com/datatorrent/bufferserver/util/CodecTest.java

```diff
@@ -22,8 +22,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.testng.annotations.Test;
-import static org.testng.Assert.*;
 
+import static org.junit.Assert.assertEquals;
```

Comment:
Also, should we get rid of all org.testing imports and use org.junit like the rest of the Apex core?